### PR TITLE
Clarify documentation for the `destroy` method

### PR DESCRIPTION
**Pull request by mandolaerik**
SIMICS-19144

### CODE REVIEW

File: lib/1.4/dml-builtins.dml [hunk 1]
```diff
@@ -3452,8 +3452,16 @@ nil.
 <small>destroy(void \*data)</small>
 </dt><dd>
 
-This method is called when an event is removed from the event
-queue
+This method is called on any posted events when the device object is deleted.
+<!-- (and also on `SIM_event_cancel_*`, but that function is not exposed for
+custom events so no need to mention) -->
```
Comment (lwaern-intel) [inline lib/1.4/dml-builtins.dml:3457]: Heh. Double-check that nothing breaks and that this comment doesn't show up in the rendered documentation.

**Review by lwaern-intel — APPROVED** [review]
Looks good, I just want you to double-check that the resulting documentation looks good if you haven't already.

**Comment by syssimics** [comment]
Verification [#10410703](https://teamcity01-ir.devtools.intel.com/viewLog.html?buildId=10410703): pass

File: lib/1.4/dml-builtins.dml (hunk 1 above)
Comment (mandolaerik) [reply lib/1.4/dml-builtins.dml:3457]: The comment is still there in the generated HTML, but does not get rendered.

**Pull request merged**